qcodes_contrib_drivers/drivers/Attocube/ANC350.py

```diff
@@ -0,0 +1,510 @@
+                           label="Actuator Name",
+                           get_cmd=self._get_actuator_name)
+
+        self.add_parameter("capacitance",
+                           label="Capacitance",
+                           get_cmd=self._get_capacitance,
+                           unit="nF")
+
+        if self._parent._version_no >= 4:
+            voltage_get: Optional[Callable] = self._get_voltage
+        else:
+            voltage_get = None
+
+        self.add_parameter("voltage",
+                           label="Voltage",
+                           get_cmd=voltage_get,
+                           set_cmd=self._set_voltage,
+                           vals=vals.Numbers(0, 70),
+                           unit="V")  # Internal resolution: mV (0.001)
+
+        self.add_parameter("output",
+                           label="Output",
+                           val_mapping={False: 0,
+                                        True: 1,
+                                        "off": 0,
+                                        "on": 1,
+                                        "auto": 2},
+                           get_cmd=self._get_output,
+                           set_cmd=self._set_output)
+
+        # Set actual unit (either mm or m°) to positional parameters
+        self._update_position_unit()
+
+    # Version 3
+    # ---------
+    def single_step(self, backward: Optional[Union[bool, str, int]] = None) -> None:
+        """
+        Triggers a single step in desired direction.
+
+        Args:
+            backward: Step direction forward (False) or backward (True). Beside True/False, you can
+                      set the direction to "forward"/"backward" or +1/-1 (default: forward or False)
+        """
+        backward = self._map_direction_parameter(backward)
+
+        self._parent._lib.start_single_step(self._parent._device_handle, self._axis, backward)
+
+    def multiple_steps(self, steps: int) -> None:
+        """
+        Performs multiple steps. The direction depends on the sign (+: forward, -: backward)
+
+        Args:
+            steps: Number of steps to move. The sign indicates the moving direction (+: forward,
+                   -: backward)
+        """
+        backward = (steps < 0)
+        
+        for i in range(abs(steps)):
+            self.single_step(backward)
+
+    def start_continuous_move(self, backward: Optional[Union[bool, str, int]] = None) -> None:
+        """
+        Starts continuous motion in forward or backward direction.
+        Other kinds of motion are stopped.
+
+        Args:
+            backward: Step direction forward (False) or backward (True). Beside True/False, you can
+                      set the direction to "forward"/"backward" or +1/-1 (default: forward or False)
+        """
+        backward = self._map_direction_parameter(backward)
+
+        self._parent._lib.start_continuous_move(self._parent._device_handle, self._axis, True,
```

Comment:
I suppose choosing True and False for start argument is also intentional and under control because the header file says start (1) and stop (0)

Reply:
I'm not quite sure, what you are telling me with that?

Do you want me to replace the start- and stop-functions with a parameter that is set to True for starting and False for stopping?

Reply:
It is fine to have method for this purpose.
The confusion is coming from the header file, when it says start (1) and stop (0). To me, it looks like that 1 is interpreted as start in the DLL, while you used True for the same purpose. So, If you say using True and False is the correct approach, it is completely fine.

Reply:
Oh, ok. That is no problem. When you take a look at the `start_continuous_move`-function of `ANC350v3Lib`, you will see that the boolean parameter `start` is converted into an `int32` before it's passed to the DLL-function. So, in the end, the DLL-function is called with 0/1 instead of True/False.